src/mcts/search.cc

```diff
@@ -518,16 +521,27 @@ bool SearchWorker::IsSearchActive() const {
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 void SearchWorker::InitializeIteration(
     std::unique_ptr<NetworkComputation> computation) {
-  nodes_to_process_.clear();
+  // If last iteration was out of order eval, continue gathering batch.
```

Comment:
Maybe slightly more detailed would be "If last iteration was {interrupted,paused} by OOO eval, resume gathering the [same][old] batch"

Reply:
No longer relevant as out-of-order eval is now inplace.